### DISCUSSION TITLE
Grafana daily analysis question

### DISCUSSION THREAD

**Discussion by johnny2678** [tagyoureit/nodejs-poolController#513] [Q&A]
This has been bugging me for a while but I never figured out a solution so I thought I would ask here

Take the `Fan % Range` graph below... graph is set up to show daily high/low/mean for the pi hat fan.  Since it's grouped by 1D, I would expect that at midnight each day, a new bar would be created and updated until the following midnight, etc.

If this screenshot was taken at 12:15am, then no complaints from me.  But it was taken at 8:15pm and you can see a new daily bar has just been created.

![image](https://user-images.githubusercontent.com/8219743/176680152-8e9108da-ad11-4169-9ccc-d14c43bdc819.png)

Except, in my setup a new daily bar is created promptly at 8pm (7pm when it's not DST).  I'm on ET so that corresponds to UTC - 4 during DST.  I have the dashboard settings set to browser time, which you can see is set to EDT.  I have also tried to explicitly set dashboard settings to EDT and global grafana settings to EDT with no change.

![image](https://user-images.githubusercontent.com/8219743/176680130-df2f809d-d37e-414c-a5b3-83b976dad805.png)

How do I get grafana to start grouping by day starting a midnight instead of 7/8pm?

**Comment by rstrouse** [comment]
Select the Timezone in your query.  It will be right under the GROUP BY clause

**Reply by johnny2678** [reply]
have tried this too with no success, though admittedly it's not in this query.  Will add and take another screenshot right after 8pm tonight.

oddly enough, the latest daily data point is 6/29 (see legend).  Today is 6/30.

![image](https://user-images.githubusercontent.com/8219743/176717933-86da26ff-b844-4651-af0a-77fc16ee60fc.png)
